host_core/lib/host_core/host.ex

```diff
@@ -46,6 +46,7 @@ defmodule HostCore.Host do
     friendly_name = HostCore.Namegen.generate()
 
     Logger.info("Host #{opts.host_key} (#{friendly_name}) started.")
+    Logger.info("Host issuer public key: #{opts.cluster_key}")
```

Comment:
I was going to call for a clarification in language here, but if you search `issuer` on our docsite it comes to https://wasmcloud.dev/app-dev/secure/clusterkeys/ so this is good